scripts/find-dead-code.py

```
@@ -53,21 +105,21 @@ def check_orphaned_modules() -> int:
-        )
+        imported = any(pattern in all_source for pattern in module_reference_patterns(module_path))
         if not imported:
             orphaned.append(str(f))
 
     if orphaned:
         print(f"{RED}  Found {len(orphaned)} orphaned module(s):{RESET}")
         for path in orphaned:
-            print(f"    {RED}✗{RESET} {path}")
+            print(f"    {RED}{FAIL}{RESET} {display_path(path)}")
+        print(f"    {CYAN}Review checklist:{RESET}")
+        print(f"      - Search tests/docs for the module name")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)